provisioning/windows-provision.ps1

```diff
@@ -65,6 +65,9 @@ Function AddToPathEnv($path) {
     Set-ItemProperty -Path 'Registry::HKEY_LOCAL_MACHINE\System\CurrentControlSet\Control\Session Manager\Environment' -Name PATH -Value $newPath | Out-Null
 }
 
+# Requried by testing libraries such as playwright
+Install-WindowsFeature Server-Media-Foundation

```

Comment:
I'm not 100% convinced this is the right place for it, the [documentation](https://support.microsoft.com/en-us/topic/media-feature-pack-list-for-windows-n-editions-c1c6fffa-d052-8338-7a79-a4bb980a700a) on the feature states that some hardware drivers might need re-installing which I'm not sure would happen after this script is run.
Happy to be told otherwise or guided on how best to do it, or to just not do it in general, as the tests appear fine other than the warning appearing at the moment but it might limit the types of tests being written depending on what functionality is required by the Playwright testing framework.